genai-perf/genai_perf/parser.py

```diff
@@ -1427,10 +1444,8 @@ def add_cli_options_to_config(
     config.input.sessions.turns.stddev = args.session_turns_stddev
 
     # Output
-    config.output.artifact_directory = args.artifact_dir
     # config.output.checkpoint_directory - There is no equivalent setting in the CLI
```

Comment:
Unrelated but seems like this comment is not used? Can we clean it up?

Reply:
This comment is indicating that we have a field (the ability to set the checkpoint directory) available in the config file, that is not present in the CLI.